nemo/utils/exp_manager.py

```diff
@@ -439,6 +442,85 @@ def on_validation_batch_end(self, trainer, pl_module, outputs, batch, batch_idx)
         self._on_batch_end("validation_step_timing in s", trainer, pl_module)
 
 
+def configure_onelogger(cfg: OmegaConf, trainer: Optional[pl.Trainer] = None) -> None:
+    """Configure OneLogger v2 callback with training telemetry and exporters.
+
+    Args:
+        cfg: The configuration object
+        trainer: Optional trainer instance for checkpoint configuration
+    """
+    try:
+        from nv_one_logger.exporter.wandb_exporter import WandbExporter
```

Comment:
Saju: we need to change this import to be the config based exporter import. so we could change the exporter destination by simple onelogger package upgrade.